tstest/integration/vms/vms_test.go

```diff
@@ -459,6 +519,75 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 	}
 }
 
+func copyBinaries(t *testing.T, conn *ssh.Client) {
```

Comment:
seems like this should be passed something more to indicate which distro/environment type it is. Are we just getting lucky for now that all distros use systemd and use the same paths?

Reply:
Yes, we are lucky that all the main distros we support use systemd and all use the same paths.

Reply:
what about adding params and calling `t.Fatal` on unknown distros?

Reply:
There's not really that much of a problem for trying to put systemd units in a non-systemd system. It won't do anything, but the unit is smaller than a disk page so it's not really that much of an issue.